lib/rubygems/commands/signout_command.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require 'rubygems/command'
+require 'rubygems/commands/query_command'
+
+class Gem::Commands::SignoutCommand < Gem::Commands::QueryCommand
```

Comment:
I still don't understand why `Gem::Commands::QueryCommand` is used here. instead of `Gem::Command`.